src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -4262,12 +4265,12 @@ private void PUSH_Handler(Instruction instruction)
 
             if (0 == dimensions && !elementBasedUpdate || !objectIndexing)
             {
-                int fp = core.Rmem.FramePointer;
+                int fp = runtimeCore.RuntimeMemory.FramePointer;
```

Comment:
Noted. We'll see if it can be moved to a more proper container
Right now nothing has changed in FramePointer. It resided there from the beginning